### PR TITLE
For Phase2 GCP multi nodes testing

### PR DESCRIPTION
**Pull request by atsushi3110**
# TL;DR

* Environment Script to build Phase2's GCP envs
* Fixed reward-data loader's bugs associated with None key/value
* slurm launch script for the above GCP nodes
    * 13B example: ds_slurm.sh 
    * 8x7 mixtral example : 8x7_ds_slurm.sh 
    
# Quick Start to run slurm on Phase-2's GCP env.
* Set an arbitrary environment name at the line with `env_name={here}` in  `build_gcp_env.sh`
* Execute the following bash script at the log-in node.
    ```shell
    bash build_gcp_env.sh
    ```
 * pip install this repo because `setup.py` at this hot-fix branch has no `requirements.txt` dependency, while this is a dirty way.
    ```shell
    conda activate ${env_name}
    cd ${ROOT_DIR_OF_THIS_REPOSITORY}
    pip install -e .
    ```
  * Run slurm
    ```shell
    cd ${ROOT_DIR_OF_THIS_REPOSITORY}/examples/scripts
    sbatch ${slurm_launch_script_name}.sh
    ```

### CODE REVIEW

**Comment by pass4** [comment]
確認前に誤ってマージしてしまったのでrevertしました
